src/popover.css

```diff
@@ -28,32 +27,35 @@
   inset: auto;
 }
 
-/* This older `:open` pseudo selector is deprecated and support will be removed
-in a later release. */
+/* This older `:open` pseudo selector is deprecated
+   and support will be removed in a later release. */
 @supports selector([popover]:open) {
+  [popover]:not(:open) {
+    display: none;
+  }
+
   [popover]:not(.\:popover-open, dialog[open]) {
     display: revert;
   }
 
-  /* stylelint-disable selector-pseudo-class-no-unknown */
   [anchor]:is(:open) {
     inset: auto;
   }
-  /* stylelint-enable selector-pseudo-class-no-unknown */
 }
 
 @supports selector([popover]:popover-open) {
+  [popover]:not(:popover-open) {
```

Comment:
`@supports selector([popover]:not(:popover-open))` ??? you test one selector then you apply a more complex one :thinking: